src/drivers/fs-lite.ts

```diff
@@ -73,8 +73,8 @@ export default defineDriver((opts: FSStorageOptions = {}) => {
       }
       return unlink(r(key));
     },
-    getKeys() {
-      return readdirRecursive(r("."), opts.ignore);
+    getKeys(_base, { maxDepth }) {
+      return readdirRecursive(r("."), opts.ignore, maxDepth);
```

Comment:
Style: can you please change it to avoid destructure? 

(generally, I keep this style to pass all keys to inner utils but I see `readdirRecursive` changed to be recursive so only a style change here for consistency)